ArduPlane/commands_logic.cpp

```diff
@@ -943,6 +963,70 @@ void Plane::exit_mission_callback()
     }
 }
 
+bool Plane::verify_landing_vtol_approach(const AP_Mission::Mission_Command &cmd)
+{
+    switch (vtol_approach_s.approach_stage) {
+        case LOITER_TO_ALT:
+            if (verify_loiter_to_alt()) {
+                Vector3f wind = ahrs.wind_estimate();
+                vtol_approach_s.approach_direction = (degrees(atan2f(-wind.y, -wind.x)));
```

Comment:
I've gotten into the habit of naming angular vars with _deg, _cd or _rad to make units clearer
also, more parentheses than needed

Reply:
As a Lisp'er I feel compelled to point out that there is no such thing as to many parens :)